tests/paragraphcommand.js

```diff
@@ -61,6 +68,19 @@ describe( 'ParagraphCommand', () => {
 			setData( document, '<paragraph>[bar</paragraph><heading1>foo]</heading1>' );
 			expect( command.value ).to.be.true;
 		} );
+
+		it( 'has proper value when moved to element that is not a block', () => {
```

Comment:
You combined two tests here – that it reacts on changes and that it's off in a block. I'd propose splitting them to clarify what's the essential part of each of them because these are two completely separate behaviours.